src/stringGen.js

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+
+const playerNum = 2;
+
+const baseArray = ['a', 'a'];
+const idArray = baseArray.map(() => makeid());
+
+fs.writeFileSync('regIDArray.json', JSON.stringify(idArray, null, 2));
+
+
+function makeid()
+{
+    let text = "";
+    const possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+    for( let i=0; i < 5; i++ )
```

Comment:
Most places that you work at will have coding standards that require any scope to have surrounding curly braces.   Because of that, I suggest doing it here.